instantsearch-compose/src/main/java/com/algolia/instantsearch/compose/searchbox/internal/SearchBoxComposeImpl.kt

```diff
@@ -0,0 +1,25 @@
+package com.algolia.instantsearch.compose.searchbox.internal
+
+import androidx.compose.runtime.MutableState
+import com.algolia.instantsearch.compose.searchbox.SearchBoxCompose
+import com.algolia.instantsearch.core.Callback
+
+/**
+ * [SearchBoxCompose] implementation.
+ */
+internal class SearchBoxComposeImpl(
+    override val query: MutableState<String>
+) : SearchBoxCompose {
+
+    override var onQueryChanged: Callback<String?>? = null
```

Comment:
I'm a bit confused by the event handler naming convention. In your code I see the mixed usage of present and past tense naming (e.g. `onValueChange` in the `SearchBox` constructor and `onQueryChanged` here). Do you apply any convention that I don't get or is it something to clean up? (btw, I always apply past tense naming in the IS iOS)

Reply:
Compose uses present for its callbacks, while the library uses past tense.
Since these extensions are mainly for Compose, I followed its convention.

Reply:
Sounds good. We have to uniformize it in the next iteration of InstanSearch.